src/main/java/pages/AllRatingsPage.java

```diff
@@ -35,6 +32,11 @@ public AllRatingsPage(WebDriver driver) {
     protected WebElement oneRatings;
     protected int rating;
 
+    public AllRatingsPage(WebDriver driver) {
+        this.driver = driver;
+        PageFactory.initElements(driver, this);
+    }
+
```

Comment:
If adding BasePage with proper constructor, then all page objects' constructors should only call BasePage's constructor.